app/models/commentt.rb

```diff
@@ -0,0 +1,11 @@
+class Commentt < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post
+  after_save :comments_count
+
+  def commentts_count
+    post.update!(:commentts_counter)
+  end
+
+  private :commentts_count
+end
```

Comment:
- [ ] Please correct the naming error, I'm not sure if you're using `post` or `postt`